modules/ROOT/pages/_partials/sync/server-auth.adoc

```diff
@@ -1,3 +1,142 @@
-= Authentication and Authorization
+= Authentication and Authorization using Keycloak
+
+Using a Keycloak service and the link:https://www.npmjs.com/package/@aerogear/voyager-keycloak[@aerogear/voyager-keycloak] module, it is possible to add security to a {sync-server} application. `@aerogear/voyager-keycloak` provides the following features out of the box.
```

Comment:
similar here, but careful, cos I don't know if you're talking about the mobile service, or a generic keycloak? ie do you mean 'Using a {keycloak-service} service' or 'Using a {idm-name} instance'?